sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_HealthcareBatchConvenienceAsync.cs

```diff
@@ -22,48 +22,44 @@ public async Task HealthcareBatchConvenienceAsync()
             var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 
             #region Snippet:TextAnalyticsSampleHealthcareBatchConvenienceAsync
-            string document = @"RECORD #333582770390100 | MH | 85986313 | | 054351 | 2/14/2001 12:00:00 AM | CORONARY ARTERY DISEASE | Signed | DIS | \
-                                Admission Date: 5/22/2001 Report Status: Signed Discharge Date: 4/24/2001 ADMISSION DIAGNOSIS: CORONARY ARTERY DISEASE. \
-                                HISTORY OF PRESENT ILLNESS: The patient is a 54-year-old gentleman with a history of progressive angina over the past several months. \
-                                The patient had a cardiac catheterization in July of this year revealing total occlusion of the RCA and 50% left main disease ,\
-                                with a strong family history of coronary artery disease with a brother dying at the age of 52 from a myocardial infarction and \
-                                another brother who is status post coronary artery bypass grafting. The patient had a stress echocardiogram done on July , 2001 , \
-                                which showed no wall motion abnormalities , but this was a difficult study due to body habitus. The patient went for six minutes with \
-                                minimal ST depressions in the anterior lateral leads , thought due to fatigue and wrist pain , his anginal equivalent. Due to the patient's \
-                                increased symptoms and family history and history left main disease with total occasional of his RCA was referred for revascularization with open heart surgery.";
+            string document = "Subject is taking 100mg of ibuprofen twice daily";
 
-            List<string> batchInput = new List<string>()
+            var list = new List<string>();
+
+            for (int i = 0; i < 6; i++)
             {
-                document,
+                list.Add(document);
             };
 
-            AnalyzeHealthcareEntitiesOperation healthOperation = await client.StartAnalyzeHealthcareEntitiesAsync(batchInput, "en");
+            AnalyzeHealthcareEntitiesOperation healthOperation = await client.StartAnalyzeHealthcareEntitiesAsync(list, "en", new AnalyzeHealthcareEntitiesOptions() { IncludeStatistics = true } );
```

Comment:
We can have batch convenience without the IncludeStatistics.

Reply:
our healthcare samples need some refactoring as the way there are right now there is a ton of repetition and not enough clarify on the purpose of each sample. 
Issue https://github.com/Azure/azure-sdk-for-net/issues/17810 is meant for the work on the samples. I added description on how the end result should look like